blender/arm/material/cycles_nodes/nodes_input.py

```diff
@@ -103,7 +104,7 @@ def parse_attribute(node: bpy.types.ShaderNodeAttribute, out_socket: bpy.types.N
     return c.cast_value('0.0', from_type='float', to_type=out_type)
 
 
-def parse_rgb(node: bpy.types.ShaderNodeRGB, out_socket: bpy.types.NodeSocket, state: ParserState) -> vec3str:
+def parse_rgb(bnode: bpy.types.ShaderNodeRGB, out_socket: bpy.types.NodeSocket, state: ParserState) -> vec3str:
```

Comment:
Looks like a typo here, in the function you still use `node` everywhere

Reply:
fixed, don't know how this still worked